pyramid/session.py

```diff
@@ -10,6 +11,16 @@
 
 from webob.cookies import SignedSerializer
 
+try:
+    from Crypto import Random
```

Comment:
This is a RNG implemented in Python, why not defer to getting randomness from the appropriate location (/dev/urandom?)